google/dev/google_install_loader.py

```diff
@@ -202,13 +219,15 @@ def __unpack_and_run():
     with open('__startup_script__.sh', 'w') as f:
         f.write('#!/bin/bash\ncd /opt/spinnaker/install\n{command}\n'
                 .format(command=command))
+
     os.chmod('__startup_script__.sh', 0555)
     write_instance_metadata('startup-script',
                             '/opt/spinnaker/install/__startup_script__.sh')
     clear_instance_metadata('startup_command')
 
     # Now run the command (which is also the future startup script).
     p = subprocess.Popen(command, shell=True, close_fds=True)
+    p.wait()
```

Comment:
Is this a good idea? the next call is `.communicate()` which doesn't make sense since `.wait()` blocks until the process has exited.